concent_api/concent_api/settings/base.py

```diff
@@ -259,6 +260,13 @@
 # Stored in a 'bytes' array, e.g. b'\xf3\x97\x19\xcdX\xda...'
 # CONCENT_ETHEREUM_PRIVATE_KEY = ''
 
+# A global constant defining the URL of the storage server
+STORAGE_SERVER_INTERNAL_ADDRESS = ''
+
+# A global constant defining Path to a directory where verifier can store files downloaded from the storage server,
+# rendering results and any intermediate files.
+VERIFIER_STORAGE_PATH = ''
```

Comment:
These values should be comment. If someone won't set these in his settings it should fail

Reply:
Outdated, already done :+1: